sw/device/tests/closed_source/BUILD.bazel

```diff
@@ -6,33 +6,132 @@ load("@lowrisc_opentitan//rules:opentitan_test.bzl", "opentitan_functest")
+# command line argument, e.g., `--define test_hooks=<config setting name>`
+#
+# Modify this as described below to add more test hooks libraries.
+################################################################################
+# Example running the open-source AES smoketest with default test hooks:
+#   bazel test //sw/device/tests:aes_smoketest --define test_hooks=default
+#        - or -
+#   bazel test //sw/device/tests:aes_smoketest
+config_setting(
+    name = "hooks_default",
+    define_values = {
+        "test_hooks": "default",
+    },
+)
+
+# Example running the open-source AES smoketest with non-default test hooks:
+#   bazel test //sw/device/tests:aes_smoketest --define test_hooks=hooks_1
+config_setting(
+    name = "hooks_1",
+    define_values = {
+        "test_hooks": "hooks_1",
+    },
+)
+
+# TH-Step 1: Copy/Uncomment below to add an additional set of test hooks.
+# config_setting(
+#     name = "hooks_2",
+#     define_values = {
+#         "test_hooks": "hooks_2",
+#     },
+# )
 
-    # `hdrs` is a list of public header files for this library.
-    hdrs = [],
+# TH-Step 2: Copy/Uncomment the row below to add an additional set of test hooks.
+_TEST_HOOKS_DEPS = select({
+    "hooks_1": [":test_hooks_1"],
+    # "hooks_2": [":test_hooks_2"],
+    "hooks_default": [],
+    "//conditions:default": [],
+})
 
-    # `deps` is a list of dependencies for this library.  You may use
-    # any library from the main opentitan repo; just prefix the label
-    # with `@lowrisc_opentitan`.
-    #
-    # Any dependencies local to this repo can simply start with `//`.
+################################################################################
+# Example Manufacturer Test Hook Library.
+#
+# Modify this section as described below to add more test hooks libraries.
+################################################################################
+cc_library(
+    name = "test_hooks_1",
+    srcs = ["test_hooks_1.c"],
     deps = [
-        "@lowrisc_opentitan//sw/device/lib/runtime:log",
+        "@//sw/device/lib/runtime:log",
     ],
-
-    # We use alwayslink to force the symbols exported by this library to
-    # override any weak symbols provided by targets in `@lowrisc_opentitan`.
     alwayslink = True,
 )
 
+# TH-Step 3: Copy/Uncomment below to add an additional test hooks library that
+# may override the default test hooks.
+# cc_library(
+#    # The name is the label that will be used to refer to this library below,
+#    # in the single `test_hooks` library that is a dependency of the OTTF.
+#    name = "test_hooks_2",
+#
+#    # `srcs` is a list of source files and private header files.
+#    srcs = ["test_hooks_2.c"],
+#
+#    # `hdrs` is a list of public header files for the `test_hooks` library.
+#    hdrs = [],
+#
+#    # `deps` is a list of dependencies for this library.  You may use
+#    # any library from the main opentitan repo; just prefix the label
+#    # with `@lowrisc_opentitan`.
```

Comment:
Maybe:
... just prefix the label with `@//` (which is shorthand for `@lowrisc_opentitan//`)

or stick with the fully specified `@lowrisc_opentitan` everywhere in this file.

Reply:
ah good catch. meant to clean that up oops. fixed.